config/encoding_presets.yml

```diff
@@ -0,0 +1,67 @@
+---
+:templates:
+  :video:
+    :audio:
+      :codec: AAC
+      :sample_rate: '44100'
+      :channels: '2'
+      :codec_options:
+        :profile: AAC-LC
+    :video:
+      :codec: H.264
+      :codec_options:
+        ColorSpaceConversionMode: None
+        InterlacedMode: Progressive
+        Level: '3'
+        MaxReferenceFrames: '3'
+        Profile: baseline
+      :keyframes_max_dist: '90'
+      :fixed_gop: 'true'
+      :frame_rate: auto
+      :display_aspect_ratio: auto
+      :sizing_policy: ShrinkToFit
+      :padding_policy: NoPad
+    :thumbnails:
```

Comment:
I don't think `active_encode` and `hyrax-active_encode` can handle non-av outputs yet.